client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -86,5 +121,55 @@ def make_ndarray(tensor_proto):
         >>> output = make_ndarray(tensor_proto)
         >>> print(output)
     '''
+    tensor_shape = [d.size for d in tensor_proto.tensor_shape.dim]
+    size = np.prod(tensor_shape, dtype=np.int64)
```

Comment:
```suggestion
    num_elements = np.prod(tensor_shape, dtype=np.int64)
```
`size` can be ambiguous in this context (does it mean elements or bytes?)